packages/data/src/world/mq/fire_temple_mq.yml

```diff
@@ -61,6 +61,7 @@
     "Fire Temple Maze Upper": "can_hookshot && (trick(OOT_HAMMER_WALLS) || has_explosives)"
   locations:
     "MQ Fire Temple Maze Lower Chest": "true"
+    "MQ Fire Temple Maze Side Room Chest": "can_play_elegy"
```

Comment:
Are you normally supposed to bring a crate from Maze Upper for this?

Reply:
yeah, from upper. that's why it's originally located there.